sdk/eventhub/Azure.Messaging.EventHubs/samples/Sample09_ObservableEventBatch.md

```diff
@@ -0,0 +1,154 @@
+# Observable Event Data Batch
```

Comment:
These Event Hubs bits should not be included.   Please consider rebasing on `main`.

Reply:
already done.

Reply:
Thank you